[Bug]: metanode maybe null pointer painc when start.

**Describe the bug**
metanode panic because of request master failed.
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x98 pc=0x1037dd2]

goroutine 1 [running]:
github.com/cubefs/cubefs/metanode.(*MetaNode).checkLocalPartitionMatchWithMaster(0xc000612300, 0x8, 0xc000132980)
    /root/tmp/cfs/go/src/github.com/cubefs/cubefs/metanode/metanode.go:102 +0x1f2
github.com/cubefs/cubefs/metanode.doStart(0x1d78650, 0xc000612300, 0xc000132980, 0x131ee55, 0xc0005d7cf0)
    /root/tmp/cfs/go/src/github.com/cubefs/cubefs/metanode/metanode.go:161 +0x233
github.com/cubefs/cubefs/cmd/common.(*Control).Start(0xc0006123f0, 0x1d78650, 0xc000612300, 0xc000132980, 0x1bb0190, 0x0, 0x0)
```

**Environment**
develop-3.3.0.

**To Reproduce**
Steps to reproduce the behavior:

**Expected behavior**
A clear and concise description of what you expected to happen.

**Additional context**
Add any other context about the problem here.

--- a/metanode/metanode.go
+++ b/metanode/metanode.go
@@ -99,6 +99,10 @@ func (m *MetaNode) checkLocalPartitionMatchWithMaster() (err error) {
 		break
 	}
 
+	if err != nil {
+		return
+	}
+
 	if len(metaNodeInfo.PersistenceMetaPartitions) == 0 {
 		return
 	}

--- a/metanode/partition_fsmop_transaction.go
+++ b/metanode/partition_fsmop_transaction.go
@@ -161,14 +161,14 @@ func (mp *metaPartition) dentryInTx(parIno uint64, name string) uint8 {
 	return proto.OpOk
 }
 
-func (mp *metaPartition) txInodeInRb(inode uint64, newTxId string) bool {
+func (mp *metaPartition) txInodeInRb(inode uint64, newTxId string) (rbInode *TxRollbackInode) {
 
 	rbIno := mp.txProcessor.txResource.getTxRbInode(inode)
 	if rbIno != nil && rbIno.txInodeInfo.TxID == newTxId {
-		return true
+		return rbIno
 	}
 
-	return false
+	return nil
 }
 
 func (mp *metaPartition) txDentryInRb(parIno uint64, name, newTxId string) bool {

--- a/metanode/partition_op_inode.go
+++ b/metanode/partition_op_inode.go
@@ -213,15 +213,17 @@ func (mp *metaPartition) TxUnlinkInode(req *proto.TxUnlinkInodeRequest, p *Packe
 	ino := NewInode(req.Inode, 0)
 	inoResp := mp.getInode(ino)
 	if inoResp.Status != proto.OpOk {
-		if rbIno := mp.txInodeInRb(req.Inode, req.TxInfo.TxID); rbIno {
+		if rbIno := mp.txInodeInRb(req.Inode, req.TxInfo.TxID); rbIno != nil {
+			respIno = rbIno.inode
 			status = proto.OpOk
+
 			item := mp.inodeTree.Get(NewInode(req.Inode, 0))
 			if item != nil {
 				respIno = item.(*Inode)
 			}
 
 			p.ResultCode = status
-			log.LogWarnf("TxUnlinkInode: inode is already unlink before, req %v, rbIno %v", req, respIno)
+			log.LogWarnf("TxUnlinkInode: inode is already unlink before, req %v, rbIno %v, item %v", req, respIno, item)
 			return nil
 		}
 

